templates/main-page.html

```diff
@@ -11,7 +11,7 @@ <h1 class="font-bold font-slab text-white text-6xl text-shadow-md tracking-wide"
                 <h2 class="font-sans font-light text-white text-shadow-md tracking-wide my-1">{{_('main_subtitle')}}</h2>
             </div>
             <div class="flex-none">
-                <a href="/hedy"><button class="green-btn text-white px-8 py-4">{{_('try_it')}}</button></a>
+                <a href="/hedy" class="green-btn text-white px-8 py-4 no-underline">{{_('try_it')}}</a>
```

Comment:
Could we re-write this to a button with an `onclick`? I find it confusing to style an `a` element as a button and behave like a button but it isn't an actual button. This is what we use in most places and it would be nice to be a bit consistent (however, it wasn't correct your change either).

Reply:
I think I can do that. It would be better too, because keyboard users might be tempted to use the space bar to active a button, but that doesn't work with links.